pghoard/restore.py

```diff
@@ -74,10 +74,12 @@ def create_recovery_conf(dirpath, site, *,
 
 
 def print_basebackup_list(basebackups, *, caption="Available basebackups"):
-    print(caption)
-    fmt = "{name:40}  {size:>10}  {time:20}  {meta}".format
-    print(fmt(name="Basebackup", size="Size", time="Start time", meta="Metadata"))
-    print(fmt(name="-" * 40, size="-" * 10, time="-" * 20, meta="-" * 12))
+    print(caption, "\n")
+    fmt = "{name:40}  {size:>10}  {time:20}  {pg_ver:6}  {start_wal:24}  {orig_size:12}  {comp:6}  {meta}".format
```

Comment:
output is now *at least* 130 characters wide.  split to two lines instead and/or don't show all of this unconditionally